lib/qesapdeployment.pm

```diff
@@ -480,4 +485,50 @@ sub qesap_ansible_script_output {
     return $output;
 }
 
+=head3 qesap_create_aws_credentials
+
+    Creates a AWS credentials file as required by QE-SAP Terraform deployment code.
+=cut
+
+sub qesap_create_aws_credentials {
+    my ($key, $secret) = @_;
+    my %paths = qesap_get_file_paths();
+    my $credfile = script_output q|awk -F\" '/aws_credentials/ {print $2}' | . $paths{qesap_conf_trgt};
+    save_tmp_file('credentials', join(' ', "[default]\naws_access_key_id", '=', $key,
+            "\naws_secret_access_key", '=', $secret, "\n"));
+    assert_script_run 'mkdir -p ~/.aws';
+    assert_script_run 'curl ' . autoinst_url . "/files/credentials -o $credfile";
+    assert_script_run "cp $credfile ~/.aws/credentials";
+}
+
+=head3 qesap_create_aws_config
+
+    Creates a AWS config file in ~/.aws as required by the QE-SAP Terraform & Ansible deployment code.
+=cut
+
+sub qesap_create_aws_config {
+    my %paths = qesap_get_file_paths();
+    my $region = script_output q|awk -F\" '/aws_region/ {print $2}' | . $paths{qesap_conf_trgt};
+    save_tmp_file('config', "[default]\nregion = $region\n");
+    assert_script_run 'mkdir -p ~/.aws';
+    assert_script_run 'curl ' . autoinst_url . "/files/config -o ~/.aws/config";
+}
+
+=head3 qesap_remote_hana_public_ips
+
+    Return a list of the public IP addresses of the systems deployed by qesapdeployment, as reported
+    by C<terraform output>. Needs to run after C<qesap_execute(cmd => 'terraform');> call.
+
+=cut
+
+sub qesap_remote_hana_public_ips {
+    my $prov = get_required_var('PUBLIC_CLOUD_PROVIDER');
+    $prov = uc($prov) eq 'EC2' ? 'aws' : (uc($prov) eq 'GCE' ? 'gcp' : $prov);
```

Comment:
Is it needed? We have https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/ab18a6421250ba5f7f6a860c02961b273c415747/lib/qesapdeployment.pm#L221 to avoid the translation.

Reply:
Oh, forgot about that. I'll drop the line.

Reply:
This did not work: https://openqaworker15.qa.suse.cz/tests/105671#step/deploy/21

I'm adding the assignment with the ternary operator again.

Reply:
Found a way to remove the line. Done.